aws/resource_aws_config_authorization.go

```diff
@@ -0,0 +1,123 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsConfigAuthorization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsConfigAuthorizationPut,
+		Read:   resourceAwsConfigAuthorizationRead,
+		Delete: resourceAwsConfigAuthorizationDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"account_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateAwsAccountId,
+			},
+			"region": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceAwsConfigAuthorizationPut(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).configconn
+
+	accountId := d.Get("account_id").(string)
+	region := d.Get("region").(string)
+
+	req := &configservice.PutAggregationAuthorizationInput{
+		AuthorizedAccountId: aws.String(accountId),
+		AuthorizedAwsRegion: aws.String(region),
+	}
+
+	_, err := conn.PutAggregationAuthorization(req)
+	if err != nil {
+		return fmt.Errorf("Error creating authorization: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s:%s", accountId, region))
+	return resourceAwsConfigAuthorizationRead(d, meta)
+}
+
+func resourceAwsConfigAuthorizationRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).configconn
+
+	accountId, region, err := resourceAwsConfigAuthorizationParseID(d.Id())
+	if err != nil {
+		return err
+	}
+
+	d.Set("account_id", accountId)
+	d.Set("region", region)
+
+	res, err := conn.DescribeAggregationAuthorizations(&configservice.DescribeAggregationAuthorizationsInput{})
```

Comment:
This call does not currently handle `NextToken` so it can incorrectly report errors/missing resources in large or separate API responses.